dask/dataframe/io/csv.py

```diff
@@ -393,11 +385,14 @@ def text_blocks_to_pandas(
             enforce,
             kwargs,
         ),
-        label=label,
+        parts,
+        meta=head,
+        divisions=(None,) * (len(blocks) + 1),
+        label="read-csv-",
```

Comment:
```suggestion
        label="read-csv",
```
`from_map`'s `name` is built with the f-string `f"{label}-{token}"` so perhaps the trailing `-` is redundant